rule.go

```diff
@@ -77,8 +81,51 @@ func (cc *Conn) GetRule(t *Table, c *Chain) ([]*Rule, error) {
 	return rules, nil
 }
 
-// AddRule adds the specified Rule. See also
-// https://wiki.nftables.org/wiki-nftables/index.php/Simple_rule_management
+// GetRuleHandle returns a specific rule's handle. Rule is identified by Table, Chain and RuleID.
+func (cc *Conn) GetRuleHandle(t *Table, c *Chain, ruleID uint32) (uint64, error) {
+	conn, err := cc.dialNetlink()
+	if err != nil {
+		return 0, err
+	}
+	defer conn.Close()
+	if ruleID == 0 {
+		return 0, fmt.Errorf("Rule's RuleID cannot be 0")
```

Comment:
here and below: don’t capitalize error strings: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
done